bankroll/converter/modelconverter.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, cast, List, Optional, Sequence, Tuple, Union
+
+try:
+    import pandas as pd
```

Comment:
To fix the [build failure](https://circleci.com/gh/bankroll-py/bankroll-model/19):
```suggestion
    import pandas as pd # type: ignore
```

Reply:
✅